src/App.css

```diff
@@ -16,6 +16,11 @@ body {
   align-items: center;
 }
 
+.navList .link {
+  text-decoration: none;
+  color: rgba(0, 0, 0, 0.6);
+}
+
 .BooksDisplay {
```

Comment:
Great job with your design so far, however, there are just few adjustments to be made 😃. Kindly ensure to improve upon the following:
- [ ] Typeface of book titles should be _RobotoSlab-Bold_, and also implement the corresponding font-size and color as specified in the Zeplin. Hint: Simply click on the specific text to display its properties on a right sidebar
- [ ] Current chapter text should have typeface as _RobotoSlab-Light_, size should be _0.813rem_ and color should be _#121212_ as can be seen in the template on Zeplin.
- [ ] Properties of all other fonts should also correspond as specified in the Zeplin template (**Type, size and color**) 
- [ ] The percentage statistics found in the middle should be aligned vertically
- [ ] Each book entry should have thine grey rounded border and appropriate spacing separating it from previous entry.
- [ ] The form inputs should have spacing corresponding to the one found in the Zeplin
- [ ] Replace the text _Profile image_ on the top right with the appropriate user icon.


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/191832407-b7a3cdad-7607-4001-9a0f-360fe8c7a78c.png)


**The Zeplin design:**
![image](https://user-images.githubusercontent.com/12745474/191832604-be0376a8-3b3c-4d5e-aefa-446c69e1cdbc.png)